src/renderer/component/publishForm/view.jsx

```diff
@@ -1,884 +1,574 @@
-  }
 
-  onFileChange() {
-    const { mode } = this.state;
-    if (this.refs.file.getValue()) {
-      this.setState({ hasFile: true });
-      if (!this.state.customUrl && mode !== 'edit') {
-        const fileName = this._getFileName(this.refs.file.getValue());
-        this.nameChanged(fileName);
-      }
-    } else {
-      this.setState({ hasFile: false });
-    }
+    return "";
   }
 
-  _getFileName(fileName) {
-    const path = require('path');
-    const extension = path.extname(fileName);
-
-    fileName = path.basename(fileName, extension);
-    fileName = fileName.replace(regexInvalidURI, '');
-    return fileName;
+  checkIsFormValid() {
+    const { name, nameError, title, bid, bidError, tosAccepted } = this.props;
+    return name && !nameError && title && bid && !bidError && tosAccepted;
   }
 
-  getNameBidHelpText() {
-    const { prefillDone, name, uri } = this.state;
-    const { resolvingUris } = this.props;
-    const claim = this.claim();
+  renderFormErrors() {
+    const { name, nameError, title, bid, bidError, tosAccepted } = this.props;
 
-    if (prefillDone) {
-      return __('Existing claim data was prefilled');
-    }
-
-    if (uri && resolvingUris.indexOf(uri) !== -1 && claim === undefined) {
-      return __('Checking...');
-    } else if (!name) {
-      return __('Select a URL for this publish.');
-    } else if (!claim) {
-      return __('This URL is unused.');
-    } else if (this.myClaimExists() && !prefillDone) {
+    if (nameError || bidError) {
+      // There will be inline errors if either of these exist
+      // These are just extra help at the bottom of the screen
       return (
-        <span>
-          {__('You already have a claim with this name.')}{' '}
-          <Link label={__('Edit existing claim')} onClick={() => this.handleEditClaim()} />
-        </span>
-      );
-    } else if (claim) {
-      const topClaimValue = this.topClaimValue();
-      if (topClaimValue === 1) {
-        return (
-          <span>
-            {__(
-              'A deposit of at least one credit is required to win "%s". However, you can still get a permanent URL for any amount.',
-              name
-            )}
-          </span>
-        );
-      }
-      return (
-        <span>
-          {__(
-            'A deposit of at least "%s" credits is required to win "%s". However, you can still get a permanent URL for any amount.',
-            topClaimValue,
-            name
-          )}
-        </span>
-      );
+        <div className="card__subtitle form-field__error">
+          {nameError && <p>{__("The URL you created is not valid.")}</p>}
+          {bidError && <p>{__("You don't have enough credits for the bid amount you chose.")}</p>}
+        </div>
+      )
     }
-    return '';
-  }
 
-  closeModal() {
-    this.setState({
-      modal: null,
-    });
+    return (
+      <div className="card__subtitle form-field__error">
+        {!title && <p>A title is required</p>}
```

Comment:
`__()`

Reply:
Also, should these be inside `<p>`?

Reply:
I guess a `<div>` is probably better since this isn't a paragraph.